### PR TITLE
[ZH] Add `Metadata`, Update `Ranking criteria`

### PR DESCRIPTION
**Pull request by YumeMuzi**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Ranking_criteria/Metadata/zh.md [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+## 常规
+
+### 规定
+
+- **歌曲元数据（歌曲信息）必须参考[主要元数据来源](/wiki/Beatmap/Primary_metadata_source)。** 如果符合 RC 标准，也可以使用相较于来源修改过的元数据。如果任何主要元数据来源都不可信，请使用最常见或容易辨识的。
+- **谱面中所有难度的 `Title`、`Artist`、`Tag`、`Source`、`BeatmapSetID` 字段必须相同。**
+- **若艺术家或歌曲名字段长度超过了限制（81 个字符），则必须缩短。**
+  - 从略去附加的标记开始。
+    - 当艺术家字段含有很长的、使用 `CV` 格式[^character-voice-actor]的艺术家，可以将虚拟角色的姓名略去，仅展示`配音演员`的姓名。
+  - 如果略去标记还是过长，请使用 `...` 在合理的位置略去之后的字段。
```
Comment (CloneWith) [inline wiki/Ranking_criteria/Metadata/zh.md:12]:
```suggestion
  - 如果略去标记还是过长，请使用 `...` 在合理的位置略去字段之后的内容。
```

这里侧重于略去字段多出来的内容

File: wiki/Ranking_criteria/Metadata/zh.md [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+
+- **如果以下 Unicode 符号子集可以罗马化时，应该在前后添加空格：**
+  
+  - [箭头补充-A (Supplemental Arrows-A)](https://en.wikipedia.org/wiki/Supplemental_Arrows-A)、[箭头补充-B (Supplemental Arrows-B)](https://en.wikipedia.org/wiki/Supplemental_Arrows-B)、[杂项符号和箭头 (Miscellaneous Symbols and Arrows)](https://en.wikipedia.org/wiki/Miscellaneous_Symbols_and_Arrows)
+  - [装饰符号 (Dingbats)](https://en.wikipedia.org/wiki/Dingbats_(Unicode_block))
+  - [杂项符号 (Miscellaneous Symbols)](https://en.wikipedia.org/wiki/Miscellaneous_Symbols)
+  
+  这不适用于艺术家故意在符号周围不使用空格的情况。比如，`。✰302？ionwan2go✰。`应该罗马化成`.*302?ionwan2go*.`，而不是`. * 302 ? ionwan2go * .`。其他字符集也应该根据具体情况处理。
+  
+- **在罗马音字段，特殊符号应被替换成样式最相近、可以打印的等效 ASCII 字符，或者直接删除。** 除了以下列出的字符之外的特殊字符应该具体情况被替换或者删除。如果替换或者删除均可行，在上下文中使用同一种方法处理。
```
Comment (CloneWith) [inline wiki/Ranking_criteria/Metadata/zh.md:53]:
```suggestion
- **如果以下 Unicode 符号子集可以罗马化时，应该在前后添加空格：**
  - [箭头补充-A (Supplemental Arrows-A)](https://en.wikipedia.org/wiki/Supplemental_Arrows-A)、[箭头补充-B (Supplemental Arrows-B)](https://en.wikipedia.org/wiki/Supplemental_Arrows-B)、[杂项符号和箭头 (Miscellaneous Symbols and Arrows)](https://en.wikipedia.org/wiki/Miscellaneous_Symbols_and_Arrows)
  - [装饰符号 (Dingbats)](https://en.wikipedia.org/wiki/Dingbats_(Unicode_block))
  - [杂项符号 (Miscellaneous Symbols)](https://en.wikipedia.org/wiki/Miscellaneous_Symbols)

  这不适用于艺术家故意在符号周围不使用空格的情况。比如，`。✰302？ionwan2go✰。`应该罗马化成`.*302?ionwan2go*.`，而不是`. * 302 ? ionwan2go * .`。其他字符集也应该根据具体情况处理。

- **在罗马音字段，特殊符号应被替换成样式最相近、可以打印的等效 ASCII 字符，或者直接删除。** 除了以下列出的字符之外的特殊字符应该具体情况被替换或者删除。如果替换或者删除均可行，在上下文中使用同一种方法处理。
```

格式小修复

File: wiki/Ranking_criteria/Metadata/zh.md [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+
+### 准则
+
+- **对于[同人圈](https://baike.baidu.com/item/%E5%90%8C%E4%BA%BA/)，你应该使用以下任一选项作为艺术家：**
+  
+  - `圈名`
+  - `邀请参与歌曲制作的知名成员`
+  - `圈名 feat. 同人圈成员/外部贡献者/歌手`
+  
+  歌曲的外部贡献者通常列在致谢名单中，专门为这首歌列出的其他同人圈成员也应包括在内。至于参与歌曲制作的成员是否足够知名，可以成为唯一的艺术家，应该根据具体情况处理。
```
Comment (CloneWith) [inline wiki/Ranking_criteria/Metadata/zh.md:101]:
```suggestion
- **对于[同人圈](https://baike.baidu.com/item/%E5%90%8C%E4%BA%BA/)，你应该使用以下任一选项作为艺术家：**
  - `圈名`
  - `邀请参与歌曲制作的知名成员`
  - `圈名 feat. 同人圈成员/外部贡献者/歌手`

  歌曲的外部贡献者通常列在致谢名单中，专门为这首歌列出的其他同人圈成员也应包括在内。至于参与歌曲制作的成员是否足够知名，可以成为唯一的艺术家，应该根据具体情况处理。
```

File: wiki/Ranking_criteria/Metadata/zh.md [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+  - **可以将艺术家全部列出，并使用 `,`、`&`、`x`、`/` 等作为分隔符。**
+  - **如果被组合的艺术家字段过长，并且没有官方艺术家团体名符合条件，也可以使用描述性的艺术家绰号。** 比如，`pippi, Mocha, Yuzu, Mani & Mari, Aiko, Alisa, Chirou, Taikonator, HitCircle, Fruit, Tama`  应该改为 `osu! cast`。如果还是不符合标准，使用 `Various Artists` 作为替代。
+- **如果动漫角色的配音演员被标为某首歌的歌手，并且歌手使用了动漫角色的身份演唱，可以使用 `角色名 (CV: 配音演员姓名)` 的格式。**
+
+## 歌曲名
+
+### 规定
+
+- **当歌曲由两首或更多歌曲制作而成，你必须使用以下任一选项作为歌曲名：**
+  - 将所有原曲歌曲名列出，并使用 `,`、`&`、`x`、`/ 作为分隔符。
```
Comment (CloneWith) [inline wiki/Ranking_criteria/Metadata/zh.md:116]:
```suggestion
  - 将所有原曲歌曲名列出，并使用 `,`、`&`、`x`、`/` 作为分隔符。
```

File: wiki/Ranking_criteria/Metadata/zh.md [hunk 5]
```diff
@@ -0,0 +1,274 @@
...
+    - 歌曲混搭也可以使用 `vs.` 来分隔不同的原曲歌曲名。
+  - 创建一个自定义的，可以概括所有原曲内容的歌曲名。比如对于 `Pippi` 制作的[歌曲混编](/wiki/Beatmap/Song_compilation)，可以使用 `Pippi's Original Songs Compilation` 作为歌曲名。
+
+### 标记
+
+#### 规定
+
+- **如果一首歌曲的非官方版本在内容、韵律或长度上都与官方版本相契合，则可以将其考虑为官方版本，并添加合适的标记。这只适用于歌曲音频几乎无法与官方版本的音频分辨开时。**
```
Comment (CloneWith) [inline wiki/Ranking_criteria/Metadata/zh.md:124]:
```suggestion
- **如果一首歌曲的非官方版本在内容、韵律或长度上都与官方版本相契合，则可以将其考虑为官方版本，并添加合适的标记。这只适用于歌曲音频几乎无法与官方版本的音频分辨开的情况。**
```

适用于...情况用在这里较好

File: wiki/Ranking_criteria/Metadata/zh.md [hunk 6]
```diff
@@ -0,0 +1,274 @@
...
+    - 如果歌曲含有人造语言，需要在标签中添加 `conlang`，并将这种人造语言的名称作为语言标签。
+    - 如果歌曲中的歌词没有实际意义，语言标签可填可不填。
+    - 如果曲风和语言并不容易辨识，例如含有多语言多曲风的歌曲，则应尽量填写容易辨识出的部分。
+  - **如果歌曲属于重混音、编辑、翻唱等，不能在其他字段中填写原曲的歌曲名和艺术家姓名。**
+
+### 准则
+
+- **重混音、重编曲或歌曲混搭，应该在标签中列出所有所使用原曲以及改编后歌曲的曲风。** 如果一首 `Anime` 歌曲被重混音成了 `Electronic` 风格，则需都填上。
+- **当符合条件时，标签应该包含以下内容：**
+  - **在权衡利弊下，被略去而未在主要的字段中使用的艺术家、歌曲名、来源元数据T。**
```
Comment (CloneWith) [inline wiki/Ranking_criteria/Metadata/zh.md:211]:
```suggestion
  - **在权衡利弊下，被略去而未在主要的字段中使用的艺术家、歌曲名、来源元数据。**
```

File: wiki/Ranking_criteria/zh.md [hunk 7]
```diff
@@ -1,36 +1,59 @@
...
+
+## 目录
+
+部分上架标准有单独的文章，并且适用于所有谱面，不论游戏模式、长度或任何其他限制。
 
-**谱面上架标准 (Ranking Criteria)** 列出了[谱面](/wiki/Beatmap)在通过[谱面上架程序](/wiki/Beatmap_ranking_procedure)时必须遵守的[规定和准则](#一般术语)。
+### 概要
 
-若想修改本文中的条目，请参照 *[建议：如何提出谱面上架标准的修改案](https://osu.ppy.sh/community/forums/topics/720532)* 这篇文章，并在[谱面上架标准论坛](https://osu.ppy.sh/community/forums/87)中提出和讨论修改案。注意，若仅仅修改文中的用词、语法、组句，而不改变条目原意，则无需发帖至论坛。
+- **[简版上架标准](Simplified_ranking_criteria)** 由于这篇文章很复杂，简版上架标准简单概括了比较重要的上架标准。请注意，它并未包含所有情况，如果有任何疑问，还是应该查阅这篇文章。
```
Comment (CloneWith) [inline wiki/Ranking_criteria/zh.md:25]:
```suggestion
- **[简版上架标准](Simplified_ranking_criteria)：** 由于这篇文章很复杂，简版上架标准简单概括了比较重要的上架标准。请注意，它并未包含所有情况，如果有任何疑问，还是应该查阅这篇文章。
```

**Review by CloneWith — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**